<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>711</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditRcDate">
      <property name="placeholderText">
       <string>DD MMMM YYYY</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelRcAppealCode">
      <property name="text">
       <string>Appeal code</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelRcDate">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelRcTitle">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEditRcTitle">
      <property name="placeholderText">
       <string>Map content, Location, Date</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelRcGlideNumber">
      <property name="text">
       <string>Glide number</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEditRcAppealCode">
      <property name="placeholderText">
       <string>MDRNL0XX</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEditRcGlideNumber">
      <property name="placeholderText">
       <string>NL-2017000001-GIS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEditChecked1_2">
   <property name="geometry">
    <rect>
     <x>1087</x>
     <y>200</y>
     <width>96</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClientApp1_2">
   <property name="geometry">
    <rect>
     <x>1291</x>
     <y>200</y>
     <width>96</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditApproved1_2">
   <property name="geometry">
    <rect>
     <x>1189</x>
     <y>200</y>
     <width>96</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditRcTitle</tabstop>
  <tabstop>lineEditApproved1_2</tabstop>
  <tabstop>lineEditChecked1_2</tabstop>
  <tabstop>lineEditClientApp1_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
